src/subdomains/yield-machine/application/api/controllers/yield-machine.controller.ts

```diff
@@ -18,4 +18,10 @@ export class YieldMachineController {
   createTransaction(@Body() dto: TransactionInputDto): Promise<string> {
     return this.yieldMachineService.create(dto);
   }
+
+  @Get('vaults')
+  @ApiExcludeEndpoint()
+  getVaults(): Promise<string[]> {
+    return this.yieldMachineService.getVaults();
+  }
```

Comment:
Wouldn't it make sense to move this to the vault subdomain `src/integration/vault` (same as for masternodes)?

Reply:
thought that those are yield-machine vaults and therefore it makes sense to access them via yield-machine endpoint

Reply:
the idea was to show at least the count of LOCK vaults on vault-maxi statistics page, and we might want to show more stats in comparison to community projects. If so we would need the yield machine vaults.

Reply:
But then we would want all vaults of LOCK?

Reply:
are there others than yield-machine vaults? will there be? I don't know 🤷‍♂️